src/service-providers/vault-level/vault-level.interface.ts

```diff
@@ -0,0 +1,85 @@
+// dirinfo
+export type TDocumentDescriptor = [string, string, any?]
+
+export interface IBaseDatasetDescriptor {
+  name: string
+  kind: 'ws' | 'xl'
+  // time in ISO format
+  'create-time': string
```

Comment:
you can create type TDateISOString = string and use it to make this more clear

Reply:
I agree, fixed